gnat2goto/driver/tree_walk.adb

```diff
@@ -2791,6 +2794,21 @@ package body Tree_Walk is
 
    end Do_Op_Concat;
 
+   -------------------------
+   --     Do_Not_Op       --
+   -------------------------
+
+   function Do_Not_Op (N : Node_Id) return Irep is
+      boolean_value : constant Irep := Do_Expression (Right_Opnd (N));
+      Op_Kind       : constant Irep_Kind := I_Op_Not;
+      Ret           : constant Irep := New_Irep (Op_Kind);
```

Comment:
Sorry for being a bit of a pain on this, but can we use `Make_Op_Not` here? (Generally prefer the `Make_XXX` constructor-type functions over setting all attributes manually)

Reply:
Yes, didn't know about it. I'll change it.